superset/reports/notifications/slack.py

```diff
@@ -59,8 +60,11 @@ class SlackNotification(BaseNotification):  # pylint: disable=too-few-public-met
 
     type = ReportRecipientType.SLACK
 
+    # Use utils function to parse & format recipients
     def _get_channel(self) -> str:
```

Comment:
Could we add a docstring to this function which includes:

1. The comment you add added (augmented), and 
2. Mention that in the Slack SDK the term channel can refer to one or more channels (comma separated)

i.e., 

```python
def _get_channel(self) -> str:
    """
    Get the recipients channel(s).
    
    Note that the Slack SDK uses the term "channel" to refer to one or more
    channels. Multiple channels are demarcated by a comma.
    
    :returns: The comma separated list of channel(s)
    """
```